lib/ts/BufferWriter.h

```diff
@@ -319,13 +320,28 @@ class FixedBufferWriter : public BufferWriter
 
   /// Reduce extent to @a n.
   /// If @a n is less than the capacity the error condition, if any, is cleared.
-  /// This can be used to clear the output by calling @c reduce(0)
-  void
+  /// This can be used to clear the output by calling @c reduce(0). In contrast
+  /// to @c clip this reduces the data in the buffer, rather than the capacity.
+  self_type &
   reduce(size_t n)
```

Comment:
This should be `resize`, that would be more standard.

Reply:
Outside the scope of this PR.

Reply:
Right. Fixed in another PR.